test/functionalTest/cases/706_direct_https_notifications/direct_https_notifications.test

```diff
@@ -0,0 +1,213 @@
+payload='{
+    "contextElements": [
+        {
+            "type": "Room",
+            "isPattern": "false",
+            "id": "Room1",
+            "attributes": [
+            {
+                "name": "temperature",
+                "type": "float",
+                "value": "26.5"
+            }
+            ]
+        }
+    ],
+    "updateAction": "UPDATE"
+}'
+orionCurl --url "/v1/updateContext" --payload "$payload" --json
+
+echo
+echo "Accumulated data:"
+echo "================="
+curl -k https://localhost:1443/dump -s -S 2> /dev/null
+echo
+
+--REGEXPECT--
+HTTP/1.1 200 OK
+Content-Length: 135
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "subscribeResponse": {
+        "duration": "P1M",
+        "subscriptionId": "REGEX([0-9a-f]{24})",
+        "throttling": "PT5S"
+    }
+}
+1: ++++++++++++++++++++
+HTTP/1.1 200 OK
+Content-Length: 396
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "contextResponses": [
+        {
+            "contextElement": {
+                "attributes": [
+                    {
+                        "name": "temperature",
+                        "type": "float",
+                        "value": ""
+                    }
+                ],
+                "id": "Room1",
+                "isPattern": "false",
+                "type": "Room"
+            },
+            "statusCode": {
+                "code": "200",
+                "reasonPhrase": "OK"
+            }
+        }
+    ]
+}
+2: ++++++++++++++++++++
+HTTP/1.1 200 OK
+Content-Length: 396
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "contextResponses": [
+        {
+            "contextElement": {
+                "attributes": [
+                    {
+                        "name": "temperature",
+                        "type": "float",
+                        "value": ""
+                    }
+                ],
+                "id": "Room1",
+                "isPattern": "false",
+                "type": "Room"
+            },
+            "statusCode": {
+                "code": "200",
+                "reasonPhrase": "OK"
+            }
+        }
+    ]
+}
+
+Accumulated data:
+=================
+POST https://127.0.0.1:REGEX(\d+)/notify
+Content-Length: 477
+User-Agent: orion/REGEX(\d+\.\d+\.\d+.*)
+Host: 127.0.0.1:REGEX(\d+)
+Accept: application/xml, application/json
+Content-Type: application/json
+
+{
+  "subscriptionId" : "REGEX([0-9a-f]{24})",
+  "originator" : "localhost",
+  "contextResponses" : [
+    {
+      "contextElement" : {
+        "type" : "Room",
+        "isPattern" : "false",
+        "id" : "Room1",
+        "attributes" : [
+          {
+            "name" : "temperature",
+            "type" : "float",
+            "value" : "23"
+          }
+        ]
+      },
+      "statusCode" : {
+        "code" : "200",
+        "reasonPhrase" : "OK"
+      }
+    }
+  ]
+}
+=======================================
+
+--TEARDOWN--
+brokerStop CB
+accumulatorStop
```

Comment:
Done in https://github.com/telefonicaid/fiware-orion/commit/9a079862d78202b7f98c72f10a31980e8b1b1ac7